docs/getting_started.md

```diff
@@ -73,7 +73,7 @@ ever stuck, you can post a question in the
    port it will send messages through (if this bridge runs on the same
    machine you can use `localhost` as the `$HOST` name):
    
-    `$ npm start -- -r -c config/config.yaml -u "http://$HOST:$MATRIX_PORT"`
+    `$ yarn start -- -r -c config/config.yaml -u "http://$HOST:$MATRIX_PORT"`
```

Comment:
`yarn [run] <script>` doesn't require `--` for option separation, and even dumps a warning about it:
> warning From Yarn 1.0 onwards, scripts don't require "--" for options to be forwarded. In a future version, any explicit "--" will be forwarded as-is to the scripts.